docs/Models/coremltools.models.nearest_neighbors.rst

```diff
@@ -1,6 +1,6 @@
-**********
+*****************
 Nearest Neighbors
```

Comment:
What difference does this change make?

Reply:
These were `WARNING: Title overline too short` errors. They were quickly fixed. I wanted to reduce the errors that occur with every run of the script, so that I can see what the real errors are.